tests/unit/src/main/scala/tests/TestingClient.scala

```diff
@@ -360,6 +362,15 @@ class TestingClient(workspace: AbsolutePath, val buffers: Buffers)
             )
         ) {
           regenerateAndRestartScalaCliBuildSever
+        } else if (
+          params.getMessage() == WorkspaceChoicePopup
+            .choicesParams(
+              ServerCommands.ConnectBuildServer.title.toLowerCase(),
+              Nil,
+            )
+            .getMessage()
```

Comment:
NIT maybe put the `choicesMessage` into a separate method, 'cause at first glance it looks as if it covered only the `Nil` folders case.

Reply:
Done!